getSimulationPA.m

```diff
@@ -15,29 +19,70 @@
     else
         op_SP.approx = 'sp';
     end
-%     SP = testSigmaPointApp(@(phi) simulateForSP(Model.exp{s}.model,Data{s}.PA.time,phi,Data{s}.condition),xi,Model.exp{s},op_SP);
```

Comment:
ugh, some things from develop branch are included here and some are not :/ not really ideal
please set scale to lin op_SP.approx is not 'sp'